src/OpenTok/Util/Validators.php

```diff
@@ -30,6 +30,29 @@ public static function validateApiKey($apiKey)
             );
         }
     }
+
+    public static function validateForceMuteAllOptions(array $options)
+    {
+        $validOptions = [
+            'excludedStreamIds' => 'array',
+            'active' => 'boolean'
+        ];
+
+        foreach ($validOptions as $optionName => $optionType) {
+            if (isset($options[$optionName])) {
+                if (getType($options[$optionName]) !== $optionType) {
+                    throw new InvalidArgumentException('Invalid type given in options for: ' . $options[$optionName]);
+                }
+            }
+        }
+
+        if (isset($options['excludedStreamIds'])) {
+            foreach ($options['excludedStreamIds'] as $streamId) {
+                self::validateStreamId($streamId);
```

Comment:
The property name for the `POST` data expected by the [API endpoint](https://tokbox.com/developer/rest/#force_mute_session) is `excludedStreams`, not `excludedStreamIds`.